### PR TITLE
Adds Checkerboard Tiles, External Hull Plating, Hardened Reinforced Floors, & Monotile Dirty Decal

### PR DESCRIPTION
**Pull request by Flareguy**
## About the PR
What it says on the tin. Hull tiles are ported from /tg/.
Hull tiles are not pryable (even by fireaxe,) so I would suggest that mappers think carefully about where they put them.

**Media**
Content Client_fzxR8uuYTY

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added external hull plating tiles & hardened reinforced tiles. They can only be normally destroyed using bombs.

### CODE REVIEW

File: Resources/Prototypes/Tiles/floors.yml [hunk 1]
```diff
@@ -1181,3 +1209,23 @@
...
+
+#Hull tiles
+- type: tile
+  id: FloorHull
+  name: tiles-hull
+  sprite: /Textures/Tiles/hull.png
+  baseTurf: Plating
+  isSubfloor: false
+  canCrowbar: false
+  canAxe: false #You can't RCD these in SS13 apparently, so this is probably the next best thing
```
Comment (deltanedas) [inline Resources/Prototypes/Tiles/floors.yml:1221]: im pretty sure i added a thing for rcd blacklist

**Comment by deltanedas** [comment]
have you tested if space wind can remove hull plating

**Comment by Flareguy** [comment]
> have you tested if space wind can remove hull plating

Yes, it doesn't get removed (unless I was just insanely unlucky.)

File: Resources/Prototypes/Tiles/floors.yml (hunk 1 above)
Comment (Flareguy) [reply Resources/Prototypes/Tiles/floors.yml:1221]: The component appears to be named "indestructible," which is definitely not what I want here (bombs should still probably be able to destroy this.) Unless the component doesn't actually make it indestructible...

**Comment by Emisse** [comment]
what is your envisioned use for hull plating

**Comment by Flareguy** [comment]
> what is your envisioned use for hull plating

Using them in space (duh,) or in other general spaced areas. I vaugely remember TramStation getting some okay mileage out of them, if you want an example.

**Comment by Emisse** [comment]
i think you should make a reinforced floor variant with these properties

**Comment by Flareguy** [comment]
> i think you should make a reinforced floor variant with these properties

Sure, simple enough.

File: Resources/Prototypes/Tiles/floors.yml (hunk 1 above)
Comment (deltanedas) [reply Resources/Prototypes/Tiles/floors.yml:1221]: Indestructible is just for rcd as per the summary, might be worth renaming for clarity idk

**Pull request merged**